src/OpenSage.Game/Terrain/Terrain.cs

```diff
@@ -402,8 +398,7 @@ private struct BlendData
                 {
                     if (tgaFile.Header.Width != largestTextureSize)
                     {
-                        tgaImage.Mutate(x => x
-                            .Resize((int) largestTextureSize, (int) largestTextureSize, MapTextureResampler));
+                        tgaImage.Mutate(x => x.Resize((int) largestTextureSize, (int) largestTextureSize, LanczosResampler.Lanczos3));
```

Comment:
Did you intentionally change this from `Lanzos2` to `Lanzos3`? (I have no idea of the pros/cons.)

Reply:
I was reading https://en.wikipedia.org/wiki/Lanczos_resampling and there was a quote from Jim Blinn:
```
According to Jim Blinn, the Lanczos kernel (with a = 3) "keeps low frequencies and rejects high frequencies better than any (achievable) filter we've seen so far."[3]
```

And after opening an discussion on the ImageSharp GitHub discussion page i was told that a range of 3 used to be the default in earlier versions. But we can stick with 2 if you prefer that

Reply:
Nope, that's fine - just wanted to check it was intentional. If Jim Blinn says something's good then we should do that :)